homeassistant/components/suez_water/quality_scale.yaml

```diff
@@ -48,25 +50,35 @@ rules:
   # Gold
   entity-translations: done
   entity-device-class: done
-  devices: done
-  entity-category: todo
-  entity-disabled-by-default: todo
+  devices:
+    status: todo
+    comment: see https://github.com/home-assistant/core/pull/134027#discussion_r1898732463
+  entity-category:
+    status: done
+    comment: only data entities
+  entity-disabled-by-default:
+    status: todo
+    comment: price can always be disabled and yesterday usage after https://github.com/home-assistant/core/pull/131166
   discovery:
     status: exempt
     comment: api only, nothing on local network to discover services
   stale-devices:
-    status: exempt
-    comment: one device only
+    status: todo
+    comment: see devices
   diagnostics: todo
   exception-translations: todo
-  icon-translations: todo
-  reconfiguration-flow: todo
-  dynamic-devices:
+  icon-translations:
+    status: exempt
+    comment: no custom icons
+  reconfiguration-flow:
     status: exempt
-    comment: one device only
+    comment: no configuration option
+  dynamic-devices:
+    status: todo
+    comment: see devices
   discovery-update-info:
     status: exempt
-    comment: fixed api
+    comment: fixed api not discoverable
```

Comment:
I'm not sure I understood your comment. So it might be discoverable, right?

Reply:
Changed the comment a bit to make it clearer